src/ignorefile/Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: test
+test:
+	dune runtest
+
+.PHONY: build
+build:
+	dune build cmd/ignorefile.exe
+
+.PHONY: clean
+clean:
+	dune clean
```

Comment:
The Makefile should be
```
.PHONY: all build check 

all: build
	@:

build:
	@apk add opam
	@opam install . --deps-only
	@opam config exec make build

check: FORCE
	@cmd/ignorefile.exe --conf-file="$(CONF_FILE)" --target-dir="$(TARGET_DIR)" --verbose="$(VERBOSE)" --commit-range="$(COMMIT_RANGE)"

FORCE:
```

You may need to change the `check` command.

Reply:
I have been thinking how should we support config file and commit range. I think it is best for a entry program to parse config and handle commit range, and then invoke the individual checker command with correct options and arguments. The advantage is that we can easily introduce new checker.

Reply:
But some check is not about commit (e.g. you may still want to run `npm audit` even it's not within the commit?) Then we need to introduce 2 type of checkers (one for commits, one for current checkout files).